pkgs/desktops/gnome-3/extensions/draw-on-your-screen/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchgit, gettext, gnome3 }:
+
+stdenv.mkDerivation rec {
+  pname = "gnome-shell-extension-draw-on-your-screen";
+  version = "6";
```

Comment:
Please update your commit message with the new version.

Reply:
Done @veprbl